src/stores/ui.ts

```diff
@@ -139,5 +178,22 @@ export const useUiStore = defineStore('ui', {
     toggleFilters() {
       this.setFiltersState(!this.isFilterVisible)
     },
+    /**
+     * If the banner wasn't dismissed before, dismisses it and saves the new state in a cookie.
+     * @param bannerId - the id of the banner to dismiss.
+     */
+    dismissBanner(bannerId: BannerId) {
+      if (!this.dismissedBanners.includes(bannerId)) {
+        this.dismissedBanners.push(bannerId)
+        this.$nuxt.$cookies.set(
+          'uiDismissedBanners',
+          this.dismissedBanners,
+          cookieOptions
+        )
+      }
```

Comment:
```suggestion
      if (!this.dismissedBanners.includes(bannerId)) {
        return
      }
      
      this.dismissedBanners.push(bannerId)
      this.$nuxt.$cookies.set(
        'uiDismissedBanners',
        this.dismissedBanners,
        cookieOptions
      )
```

Personally prefer early returns but it doesn't matter.